src/Forms/ConfigurationForm.cs

```diff
@@ -264,6 +288,11 @@ public bool IsExpressWorkflowSelected()
             return ExpressWorkflowRadioButton.Checked;
         }
 
+        public bool IsAvsProposalSelected()
+        {
+            return QuickAvsProposalRadioButton.Checked;
+        }
+

```

Comment:
return enum from here as mentioned

public Enum ProposalTypes
{
Comprehensive,
AVS,
}

Reply:
followed format for isExpressWorkflow

Reply:
After our discussion, to support extensibility, I have converted this to Enum